src/Components/Pages/Results/Results.tsx

```diff
@@ -457,3 +458,95 @@ const CoverageResultHeadline: React.FC<{
   }
   return <span ref={headlineRef}>{headlineContent}</span>;
 };
+
+const PhoneNumberCallout: React.FC = () => {
+  const [phoneNumber, setPhoneNumber] = useState("");
+  const [showError, setShowError] = useState(false);
+  const VALID_PHONE_NUMBER_LENGTH = 10;
+
+  const { user } = useLoaderData() as {
+    user?: GCEUser;
+  };
+  const { trigger } = useSendGceData();
+  const rollbar = useRollbar();
+
+  const formatPhoneNumber = (value: string): string => {
+    // remove all non-digit characters
+    const cleaned = value.replace(/\D/g, "");
+    // limit to 10 characters
+    const limited = cleaned.slice(0, 10);
+
+    // format with parentheses and dashes e.g. (555) 666-7777
+    const match = limited.match(/^(\d{0,3})(\d{0,3})(\d{0,4})$/);
+    if (match) {
+      const [, part1, part2, part3] = match;
+      const formatted = [
+        part1 ? `(${part1}` : "",
+        part2 ? `) ${part2}` : "",
+        part3 ? `-${part3}` : "",
+      ]
+        .join("")
+        .trim();
+      return formatted;
+    }
+    return value;
+  };
+
+  const handleInputChange: React.ChangeEventHandler<HTMLInputElement> = (e) => {
+    const value = formatPhoneNumber(e.target.value);
+    setPhoneNumber(value);
+  };
+
+  const handleSubmit = () => {
+    const cleaned = phoneNumber.replace(/\D/g, "");
+    if (cleaned.length === VALID_PHONE_NUMBER_LENGTH) {
+      try {
+        trigger({
+          id: user?.id,
+          phone_number: parseInt(cleaned),
+        });
+        setShowError(false);
+      } catch (error) {
+        rollbar.error("Cannot connect to tenant platform");
```

Comment:
elsewhere for these I was using critical, so just to stay consistent:
```suggestion
        rollbar.critical("Cannot connect to tenant platform");
```